src/main/java/koksao/Main.java

```diff
@@ -0,0 +1,92 @@
+package koksao;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+
+import javax.swing.text.html.Option;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+
+
+    public static void main(String[] args) {
+        Options options = new Options();
+        options.addOption("r", "rates", true, "path to a file");
+
+        CommandLineParser clp = new DefaultParser();
+        try {
+            CommandLine cl = clp.parse(options, args);
+            if (cl.hasOption("r")) {
+                String filePath = cl.getOptionValue("r");
+                args[0] = filePath;
+            } else {
+                System.out.println("Cannot find a file");
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        Scanner scanner = new Scanner(System.in);
+        String answer = "";
+        String firstCurrency;
+        String secondCurrency;
+
+        while (!answer.equals("Exit")) {
+            System.out.println("What currency you want to convert from?");
+            firstCurrency = scanner.next();
+            System.out.println("On which?");
+            secondCurrency = scanner.next();
+            System.out.println("What amount of money?");
+            int amount = scanner.nextInt();
+
+            String line;
+            String csvSplitBy = ",";
+            try {
+                BufferedReader br = new BufferedReader(new FileReader(args[0]));
+                while ((line = br.readLine()) != null) {
+                    String[] data = line.split(csvSplitBy);
+                    if (data[0].equals(firstCurrency) && data[1].equals(secondCurrency)) {
+                        double rate = Double.parseDouble(data[2]);
+                        System.out.println(String.format("%.2f", rate * amount));
+                    }
+                    if (data[1].equals(firstCurrency) && data[0].equals(secondCurrency)) {
+                        double rate = Double.parseDouble(data[2]);
+                        System.out.println(String.format("%.2f", (1 / rate) * amount));
+
+                    }
+                }
+            } catch (FileNotFoundException e) {
+                throw new RuntimeException(e);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+            System.out.println("If you want to end program type Exit, if not type No");
```

Comment:
‘Type no’ if I do not type no, te same thing will happen. Change the message or behavior of the program.